Implement dprint and biome as a linter and formatter

bjgavilanes opened the pull request:
At 22a86b0, I set up biome with default config. The only configuration made was to enable vcs support, and do follow .gitignore.

964d89d, setup dprint, to supplement biome shortcommings. As of now, biome does provide HTML and supersets support, but is partial. Also setup formatting for yaml, docker and css-related.

9767ad3, the result of intended ci pipeline: to invoque dprint and then biome.

This can be used, even, as a git-hook strategy.

davcortez commented:
Like I said in the past PR, you must provide some instructions in the README to help new contributors understand how this works. Additionally, I want to explore the [discussion](https://github.com/orgs/Ecuador-In-Tech/discussions) area to talk about this kind of stuff before its implementation.

bjgavilanes commented:
@davcortez I totally get you. I thought it would be better if we implement this using a task runner -- whether be `make`, `just`, or another -- but might as well at least put a task/ dir with a reference script; and then, migrate those to the task runner / build system of our liking.

For the <<discussions>> point... I would advice against it. It would be another variable to care when / if we migrate from GitHub. Also, I don't know if discussions too send you and email such as issues / PR.

Anyways, brb with more commits.

bjgavilanes commented:
At 56215b7, added `@biomejs/biome` and `dprint` as development dependencies at package.json.
9c50e71, added install and development instructions at README
I did a rebase, so yeah. Git is hard, man.

bjgavilanes commented:
At last!, ready to merge, bosses.

@davcortez @IvanM9

bjgavilanes commented:
d2dadc9, fixed a linter problem related to CSS. Read for context: https://github.com/biomejs/biome/discussions/4141

Already set!

IvanM9 reviewed (approved):
Perfect, works
